sycl/test-e2e/Plugin/level_zero_queue_priority.cpp

```diff
@@ -17,19 +19,23 @@ void test(sycl::property_list Props) {
 
 int main(int Argc, const char *Argv[]) {
 
-  // CHECK: [getZeQueue]: create queue {{.*}} priority = Normal
+  // CHECK-STD: [getZeQueue]: create queue {{.*}} priority = Normal
+  // CHECK-IMM: [getZeQueue]: create queue {{.*}} priority = Normal
   test(sycl::property_list{});
 
-  // CHECK: [getZeQueue]: create queue {{.*}} priority = Normal
+  // CHECK-STD: [getZeQueue]: create queue {{.*}} priority = Normal

```

Comment:
Please either add a CHECK-IMM or a comment why it shouldn't be here.

Reply:
CHECK-IMM is not needed here precisely because of IMM recycling.
I've added a comment saying so.